Dockerfile

```diff
@@ -3,13 +3,13 @@ FROM docker.io/library/alpine:3.9@sha256:769fddc7cc2f0a1c35abb2f91432e8beecf8391
 LABEL io.openshift.s2i.scripts-url=image:///usr/libexec/s2i \
       io.openshift.s2i.assemble-user=65534:0
 
-ENV HELM_VERSION=v2.13.1 \
+ENV HELM_VERSION=v2.14.0 \
```

Comment:
this version has issues with OpenShift objects (f.e. `Route`). We should wait until `v2.14.1` is available.

Reply:
Here we go: Helm 2.14.1 released